_maps/RandomRooms/greytopia.dmm

```diff
@@ -0,0 +1,562 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/template_noop,
+/area/template_noop)
+"b" = (
+/turf/closed/wall/r_wall,
+/area/greytopia)
+"c" = (
+/obj/item/banner/greytide{
+	pixel_y = -4
+	},
+/turf/closed/wall/r_wall,
+/area/greytopia)
+"d" = (
+/obj/item/cardboard_cutout/adaptive{
+	name = "Nameless Hero";
+	icon_state = "cutout_viva";
+	pixel_y = -5
+	},
+/turf/closed/wall/r_wall,
+/area/greytopia)
+"e" = (
+/obj/structure/statue/sandstone/assistant{
+	pixel_y = -5
+	},
+/turf/closed/wall/r_wall,
+/area/greytopia)
+"f" = (
+/obj/structure/weightlifter,
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"g" = (
+/obj/structure/bed,
+/obj/item/bedsheet/grey,
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"h" = (
+/obj/structure/punching_bag,
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"i" = (
+/mob/living/simple_animal/bot/medbot/mysterious{
+	stationary_mode = 1
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"j" = (
+/obj/structure/bonfire/dense,
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"k" = (
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"l" = (
+/obj/item/target,
+/obj/item/gun/energy/laser/chameleon{
+	pixel_y = -8
+	},
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"m" = (
+/obj/structure/bed,
+/obj/item/bedsheet/grey,
+/turf/open/floor/plating{
+	icon_state = "panelscorched"
+	},
+/area/greytopia)
+"n" = (
+/turf/open/floor/plating{
+	icon_state = "platingdmg1"
+	},
+/area/greytopia)
+"o" = (
+/obj/structure/chair,
+/obj/item/device/radio/headset/headset_com{
+	flags_1 = 80;
+	flags_2 = 66;
+	name = "\proper the head of assistants's bowman headset"
+	},
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"p" = (
+/obj/structure/mirror{
+	pixel_x = 5
+	},
+/turf/closed/wall/r_wall,
+/area/greytopia)
+"q" = (
+/obj/structure/sink{
+	dir = 8;
+	pixel_x = -12
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"r" = (
+/obj/effect/decal/cleanable/blood/gibs/torso,
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"s" = (
+/obj/structure/chair{
+	dir = 4
+	},
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"t" = (
+/obj/structure/table,
+/obj/item/reagent_containers/food/snacks/salad/validsalad{
+	pixel_x = -9;
+	pixel_y = 2
+	},
+/obj/item/reagent_containers/food/snacks/salad/validsalad{
+	pixel_x = 7;
+	pixel_y = 1
+	},
+/obj/item/candle/infinite{
+	pixel_y = 9
+	},
+/turf/open/floor/plating{
+	luminosity = 2
```

Comment:
Why not a light tile instead?

Reply:
Probably me var-editing the floor instead of the light on top of it, ill have to fix that but poor Greytopia isn't even enabled in this PR

Reply:
Best to fix it anyhow dirty vars are bad